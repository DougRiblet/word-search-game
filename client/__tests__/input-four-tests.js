import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { mount } from 'enzyme';
import InputFour from '../components/Input-Four/input-four-present';

describe('Input Four', () => {
  test('Snapshot', () => {
    const InputFour_tree = renderer.create(
      <InputFour poolfour={['JEEZ']} foundfour={['EELY']} wrongfour={[]} />
    ).toJSON();
    expect(InputFour_tree).toMatchSnapshot();
  });

  test('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<InputFour poolfour={['JEEZ']} foundfour={['EELY']} wrongfour={[]} />, div);
  });

});
