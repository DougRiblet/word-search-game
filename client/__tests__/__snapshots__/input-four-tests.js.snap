// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input Four Snapshot 1`] = `
<div
  id="input-four"
>
  <form
    onSubmit={[Function]}
  >
    <input
      disabled={undefined}
      id="inputguess"
      maxLength="4"
      minLength="4"
      onChange={[Function]}
      type="text"
      value=""
    />
  </form>
</div>
`;
